### DISCUSSION TITLE
Device password is not working via knobby.local

### DISCUSSION THREAD

**Discussion by ghost** [quadule/knobby#15] [General]
Hi,
I've replaced the encoder to one that count once per 2 click and i wanted to change pulse count. As I can see in data.json in github there is no password. There is no option to set it while configuring so when i want to update settings no matterif the password field is empty or no it's not letting me update it and shows error "Incorrect password". How can I make it work?
I have it on calssic ttgo display.

**Comment by quadule** [comment]
Hi, sounds like you got this working since you closed this as resolved, but I realize that this wasn't properly documented:

The config password will be randomly generated on first boot if missing. You can see it in the serial console output over USB, or by double-clicking the knob on the settings > about screen.
